product_docs/docs/edbcloud/beta/overview/06_support.mdx

```diff
@@ -4,7 +4,7 @@ title: "Support Options"
 
 If you experience problems with EDB Cloud, you have several options to engage with EDB's Support team to get help. If you have an EDB Cloud account, you can go directly to the Support portal or the EDB Cloud portal to open a support case, or you can leave Support a message using the Support widget kCase. 
 
-If you can’t log in to your account, send us an email to cloudsupport@enterprisedb.zendesk.com[support@enterprisedb.zendesk.com. 
+If you can’t log in to your account, send us an email to cloudsupport@enterprisedb.com
```

Comment:
If you can’t log in to your account, send us an email to [cloudsupport@enterprisedb.com](cloudsupport@enterprisedb.com).